components/CarbonsAds.tsx

```diff
@@ -115,6 +115,12 @@ CarbonAds.stylesheet = {
       margin-top: 4px;
       color: rgb(100 116 139);
     }
+
+    @media (max-width: 1023px) {
```

Comment:
Why pushing this change? please remove from the PR anything that is unrelated to this feature.